wsgi/bjoern.rst

```diff
@@ -0,0 +1,226 @@
+Bjoern
+======
+
+`bjoern <https://github.com/jonashaag/bjoern>`_ is a fast And ultra-lightweight HTTP/1.1 WSGI Server for CPython2 and 3 written in C.
+It claims to be the fastest, smallest and most lightweight WSGI server.
+Features:
+
+* ~ 1000 lines of C code
+* Memory footprint ~ 600KB
+* supports Python 2 and 3
+* Single-threaded
+* supports TCP `host:port` addresses and Unix sockets
+* Full persistent connection ("*keep-alive*") support in both HTTP/1.0 and 1.1,
+  including support for HTTP/1.1 chunked responses
+
+.. note::
+
+    In a load test involving `bjoern`, `cheroot`, `gunicorn`, `waitress` and `werkzeug` ``bjoern`` (version: 3.0.0) was the clear speed winner against both a ZEO and a non-ZEO Zope instance.
+    Source: Zope/docs/wsgi.rst:
+
+Prerequisites
+-------------
+
+Bjoern uses libev and you will need to install both the library and the development header files on your box:
+
+.. code-block:: bash
+
+    $ sudo apt install libev-dev
+
+Use bjoern in our buildout
+--------------------------
+
+.. sidebar:: Build now
+
+    Run buildout for this section:
+
+    ..  code-block:: bash
+
+        buildout -c bjoern.cfg
+
+`bjoern <https://github.com/jonashaag/bjoern>`_ can be integrated using a shim package called `dataflake.wsgi.bjoern <https://dataflakewsgibjoern.readthedocs.io/>`_.
+
+You can use this package together with `plone.recipe.zope2instance` to build a bjoern based wSGI setup:
+
+.. code-block:: ini
+
+    [instance]
+    recipe = plone.recipe.zope2instance
+    user = admin:admin
+    zeo-client = on
+    zeo-address = 8100
+    shared-blob = on
+    blob-storage = ${buildout:directory}/var/blobstorage
+    eggs =
+        Plone
+        Pillow
+        wsgitraining.site
+        dataflake.wsgi.bjoern
+    wsgi-ini-template = ${buildout:directory}/templates/bjoern.ini.in
+
+In addition to adding `dataflake.wsgi.bjoern` to the `eggs` list we specify the location of our `bjoern.ini` configuration file.
+It is important to note that this file is not automatically created for us, we have to provide it ourself.
+
+In addition to the PasteDeploy entry point and the p.r.zope2instance integration, `dataflake.wsgi.bjoern`  provides facilities to create a set of Zope configuration files for bjoern with the included `mkbjoerninstance` utility.
+We will however not use this option since it is easier for us to provide a custom template for the `wsgi.ini` file to `plone.recipe.zope2instance`.
```

Comment:
```suggestion
We will however not use this option since it is easier for us to provide a custom template for the ``wsgi.ini`` file to ``plone.recipe.zope2instance``.
```

Reply:
Thank you - done in https://github.com/plone/training/pull/436/commits/b4db67ecee1b8a6044d836ed88f1097a022a6739.